### PR TITLE
Update intranet-overview.md

### PR DESCRIPTION
**Pull request by gavindavies**
'to' to 'do'.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointOnline/intranet-overview.md [hunk 1]
```diff
@@ -182,7 +182,7 @@ Depending on their background, the people in your organization may need some hel
...
+As a business owner, you may be responsible for intranet content for a specific area of your organization such as HR or operations. You may also be the site owner, the person who has the permissions in SharePoint to create and maintain the content and experiences on the site. Before you start configuring individual sites, for some ideas, look over the [SharePoint look book](https://aka.ms/sharepointlookbook). As part of your overall governance plan, you may decide that sites need to follow consistent patterns to ensure that readers do not have to spend too much time orienting themselves as the go from site to site or topic to topic in the intranet.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/intranet-overview.md:185]:
```suggestion
As a business owner, you may be responsible for intranet content for a specific area of your organization such as HR or operations. You may also be the site owner, the person who has the permissions in SharePoint to create and maintain the content and experiences on the site. Before you start configuring individual sites, for some ideas, look over the [SharePoint look book](https://aka.ms/sharepointlookbook). As part of your overall governance plan, you may decide that sites need to follow consistent patterns to ensure that readers do not have to spend too much time orienting themselves as they go from site to site or topic to topic in the intranet.
```

Comment (JohanFreelancer9) [reply SharePoint/SharePointOnline/intranet-overview.md:185]: Please change "the" to "they".

**Comment by scanum** [comment]
@MikePlumleyMSFT PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**